src/components/Navi/index.js

```diff
@@ -0,0 +1,40 @@
+import React,{ useContext } from 'react'
+import Nav from './Nav'
+import Navlinks from './Navlinks'
+import Icon from '../shared/Icon';
+import iconDir from '../icon.svg';
+import { Link } from 'react-router-dom';
+import Taglist from '../Taglist';
+import { AppContext } from '../../contexts/AppContext';
+import { ThemeContext } from '../../contexts/ThemeContext';
+
+export default function () {
+    const { state } = useContext(AppContext);
+    const { darkMode } = useContext(ThemeContext);
+    return (
```

Comment:
Add extra line between these two since there two different kinds of logic? Like on is a variable declaration/assignment statement while the other one is a return statement?